src/models/connect.js

```diff
@@ -58,4 +60,24 @@ export default class Connect {
   newAccount() {
     // this.authorize() -> this.token()
   }
+
+  async gmailAccount({ name, emailAddress, scopes, googleClientId, googleClientSecret, googleRefreshToken }) {
+    const googleOptions = {
+      name: name,
+      email_address: emailAddress,
+      provider: 'gmail',
+      settings: {
+        googleClientId: googleClientId || this.googleClientId,
+        googleClientSecret: googleClientSecret || this.googleClientSecret,
+        googleRefreshToken: googleRefreshToken
+      }
+    }
+
+    if (scopes) {
+      googleOptions.scopes = scopes;
+    }
+
+    token = await this.authorize(googleOptions);
```

Comment:
if we got the async/await route, will errors from authorize() still be surfaced? 
also, let's assign token as a const (const token = await this.authorize(googleOptions))